packages/account/src/Components/poi/idv-document-submit/idv-document-submit.jsx

```diff
@@ -103,7 +105,7 @@ const IdvDocumentSubmit = ({ handleBack, handleViewComplete, selected_country, i
             errors.document_number =
                 localize('Please enter your document number. ') + getExampleFormat(document_type.example_format);
         } else if (
-            (validation_is_enabled && (is_recurring_number || is_sequential_number)) ||
+            (!is_idv_whitelist_document_number && (is_recurring_number || is_sequential_number)) ||
             document_number === document_type.example_format
```

Comment:
Perhaps we can put all these in one variable and pass that in the `else if` statement for better readability.

Perhaps like this? 
```
const document_is_invalid = !is_idv_whitelist_document_number && (is_recurring_number || is_sequential_number)) || document_number === document_type.example_format

else if (document_is_invalid) { ... }
```